rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -277,11 +342,11 @@ def get_state(self) -> Dict[str, Any]:
 
     @DeveloperAPI
     def set_state(self, state: Dict[str, Any]) -> None:
-        """Restores all local state to the provided `state`.
+        """Restores all local state to the provided 'state'.
 
         Args:
             state: The new state to set this buffer. Can be
-                obtained by calling `self.get_state()`.
+                obtained by calling 'self.get_state()'.
```

Comment:
Any reason we are changing the backticks to single-quotes?
I think when referring to variables, args, methods, we always use backticks, no?

Also: @maxpumperla ^

Reply:
No, this is a mistake on my side! No idea how that happened.